src/NHSD.BuyingCatalogue.Ordering.Common/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using NHSD.BuyingCatalogue.Ordering.Common.Constants;
+
+namespace NHSD.BuyingCatalogue.Ordering.Common.Extensions
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection RegisterHealthChecks(this IServiceCollection services, string connectionString)
+        {
+            if (connectionString is null)
+                throw new ArgumentNullException(nameof(connectionString));
+
+            services.AddHealthChecks()
+                .AddCheck(
+                    "self",
+                    () => HealthCheckResult.Healthy(),
+                    new[] { HealthCheckTags.Live })
+                .AddSqlServer(
+                    connectionString,
+                    "SELECT 1;",
+                    "db",
+                    //HealthStatus.Unhealthy,
+                    HealthStatus.Healthy,  // database not available yet hard coding response to healthy until it is implemented
```

Comment:
Can you reformat this slightly so it's more along the lines of:

```c#
"db",

// TODO: database not available yet hard coding response to healthy until it is implemented
HealthStatus.Healthy,
```
I think this format is a bit more readable and we can track `TODO` comments using the task list in VS. (Feel free to amend the comment if you think necessary.)

Do we have a task to correct this once the database is implemented? If not, can we create one or make sure there is a note on the database task that covers this? (Assuming you have DevOps access now – if not I can do this.)

Reply:
I think i can now create tasks so i will have a go at that.